src/lib/journey/callbacks/_utilities/callback.utilities.ts

```diff
@@ -69,16 +69,50 @@ export function getInputTypeFromPolicies(policies: StringDict<unknown>): 'email'
 export function getValidationFailureParams(
   failedPolicy: PolicyRequirement | undefined,
 ): RestructuredParam[] {
-  if (failedPolicy?.policyRequirement === 'CHARACTER_SET') {
-    const params = failedPolicy?.params as {
+  if (failedPolicy?.policyRequirement === 'DICTIONARY') {
```

Comment:
This adds a few policies that were missing to the `ValidatedCreatePasswordCallback` that Stoyan found.